### PR TITLE
Format generated code using Goethe (palantir-java-format for javapoet)

### PR DESCRIPTION
**Pull request by carterkozak**
I suspect we'll need to update some generated sources to reflect the new formatting, but I'm not familiar enough with this codebase to find those outputs without running the test suite.

**Goals (and why)**:

Standardize generated source formatting, should produce code that's easier to read.

**Implementation Description (bullets)**:

https://github.com/palantir/goethe

**Testing (What was existing testing like?  What have you done to improve it?)**:

**Concerns (what feedback would you like?)**:

**Where should we start reviewing?**:

**Priority (whenever / two weeks / yesterday)**:

### CODE REVIEW

File: atlasdb-client/src/test/java/com/palantir/atlasdb/table/description/SchemaHotspottingTest.java [hunk 1]
```diff
@@ -49,7 +49,7 @@ private static Schema getHotspottingSchema() {
     }
 
     private static Schema getIgnoredHotspottingSchema() {
-        Schema ignoredHotspottingSchema = new Schema(SCHEMA_NAME, "valid.package", Namespace.DEFAULT_NAMESPACE);
+        Schema ignoredHotspottingSchema = new Schema(SCHEMA_NAME, "valid.pkg", Namespace.DEFAULT_NAMESPACE);
```
Comment (carterkozak) [inline atlasdb-client/src/test/java/com/palantir/atlasdb/table/description/SchemaHotspottingTest.java:52]: I got a chuckle out of this.

Comment (jeremyk-91) [reply atlasdb-client/src/test/java/com/palantir/atlasdb/table/description/SchemaHotspottingTest.java:52]: Wasn't expecting this... it's fine, though is this a thing we expect from the formatter?

**Review by jeremyk-91 — APPROVED** [review]
Great! Yeah, I think this will be nice :)

File: atlasdb-client/src/test/java/com/palantir/atlasdb/table/description/SchemaHotspottingTest.java (hunk 1 above)
Comment (carterkozak) [reply atlasdb-client/src/test/java/com/palantir/atlasdb/table/description/SchemaHotspottingTest.java:52]: Yep, it uses the jdk ast parser, which does some validation.

**Pull request merged**